### PR TITLE
Move Compass Runtime Agent to Kyma

### PR DESCRIPTION
**Pull request by akgalwas**
**Description**
Changes proposed in this pull request:
- Compass Runtime Agent moved to Kyma

### CODE REVIEW

File: components/compass-runtime-agent/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+#
+# Build app
+#
+
+RUN go build -v -o compass-runtime-agent ./cmd/main.go
+RUN mkdir /app && mv ./compass-runtime-agent /app/compass-runtime-agent && mv ./licenses /app/licenses
+
+FROM alpine:3.9
+LABEL source = git@github.com:kyma-incubator/compass.git
```
Comment (Szymongib) [inline components/compass-runtime-agent/Dockerfile]:
```suggestion
LABEL source = git@github.com:kyma-project/kyma.git
```

File: components/compass-runtime-agent/Makefile [hunk 2]
```diff
@@ -0,0 +1,37 @@
+IMG_NAME = compass-runtime-agent
```
Comment (Szymongib) [inline components/compass-runtime-agent/Makefile]:
```suggestion
APP_NAME = compass-runtime-agent
```

File: components/compass-runtime-agent/Makefile [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(IMG_NAME)
```
Comment (Szymongib) [inline components/compass-runtime-agent/Makefile]:
```suggestion
IMG = $(DOCKER_PUSH_REPOSITORY)$(DOCKER_PUSH_DIRECTORY)/$(APP_NAME)
```

File: components/compass-runtime-agent/Makefile (hunk 2 above)
Comment (akgalwas) [reply components/compass-runtime-agent/Makefile]: Done

File: components/compass-runtime-agent/Makefile (hunk 3 above)
Comment (akgalwas) [reply components/compass-runtime-agent/Makefile]: Done

**Pull request merged**